gridstatus/tests/test_eia.py

```diff
@@ -37,6 +37,23 @@ def _check_region_data(df):
     assert df["Interval End"].dtype == "datetime64[ns, UTC]"
     assert df.shape[0] > 0
     assert df.columns.tolist() == columns
+    
+    
+def _check_region_subba_data(df):
```

Comment:
In Pytest, only method names starting with `test_` are interpreted as tests. The other methods in this file that start with `_check` are called by `test_` methods. This method should be renamed to something like `test_region_sub_ba_data`. 

https://docs.pytest.org/en/6.2.x/goodpractices.html#test-discovery

Reply:
Btw, you said you couldn't run the tests locally because you're on Windows...have you tried using `poetry` to run the tests? You can try `poetry run pytest gridstatus/tests/test_eia.py ` to run just the `eia` tests.

Reply:
Forgetting to actually add the `test_` was a total goof on my part! I was distracted yesterday. Also, this is my first time using `poetry` so it's a little unfamiliar! I was able to run the tests and they all pass locally.